assets/js/frontend-form.js

```
@@ -976,6 +983,52 @@
+            if (value === "") {
+                return;
+            }
+
+            // Validate Twitter URL/username
+            if (!WP_User_Frontend.isValidTwitterURL(value)) {
+                $container.addClass("has-error");
+                $container.append(
+                    '<div class="wpuf-error-msg">Please enter a valid Twitter username (e.g., @username) or Twitter URL.</div>'
+                );
+            }
+        },
```

Biome (1.9.4)

[error] This property value named validateTwitterField is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named validateTwitterField


(lint/suspicious/noDuplicateObjectKeys)